styra-das-deploy/src/push-policy.js

```diff
@@ -0,0 +1,52 @@
+const request = require('request');
+
+const fetchPolicy = (styraToken, systemId) => new Promise((resolve, reject) => {
+  request({
+    uri: `https://extendaretail.styra.com/v1/policies/systems/${systemId}/policy/com.styra.envoy.ingress/rules/rules`,
```

Comment:
To make this generic, we should have the tenant or the DAS base URL be an input variable. We can default it to our extendaretail tenant.